### PR TITLE
Add link to Changelog

### PR DESCRIPTION
**Pull request by vdm**
## Problem
The log has lots of maven-release-plugin and Jenkins automatically generated commits. This makes it difficult to understand or communicate the history of real changes.

## Solution
There is a Changelog on the Confluent website. This adds a link to the existing Changelog.

##### Does this solution apply anywhere else?
- [ ] yes
- [x] no

## Test Strategy
Click on the link.

##### Testing done:
- [x] Manual tests

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,5 @@
 # Kafka Connect Elasticsearch Connector
-[![FOSSA Status](https://app.fossa.io/api/projects/git%2Bhttps%3A%2F%2Fgithub.com%2Fconfluentinc%2Fkafka-connect-elasticsearch.svg?type=shield)](https://app.fossa.io/projects/git%2Bhttps%3A%2F%2Fgithub.com%2Fconfluentinc%2Fkafka-connect-elasticsearch?ref=badge_shield)
-
+[![FOSSA Status](https://app.fossa.io/api/projects/git%2Bhttps%3A%2F%2Fgithub.com%2Fconfluentinc%2Fkafka-connect-elasticsearch.svg?type=shield)](https://app.fossa.io/projects/git%2Bhttps%3A%2F%2Fgithub.com%2Fconfluentinc%2Fkafka-connect-elasticsearch?ref=badge_shield) [Changelog](https://docs.confluent.io/kafka-connect-elasticsearch/current/changelog.html)
```
Comment (kpatelatwork) [inline README.md:2]: @vdm this link looks odd in the screenshot, could we please add Changelog on a new line? We can add something like 

Changelog for this connector can be found [here](https://docs.confluent.io/kafka-connect-elasticsearch/current/changelog.html).



image

**Comment by kpatelatwork** [comment]
@vdm Thanks a lot for your changes, do you mind targeting the changes to 11.1.x branch?

**Comment by vdm** [comment]
Rebased to 11.1.x and incorporated requested change. Thank you @kpatelatwork for your review.

**Review by kpatelatwork — APPROVED** [review]
Very nice @vdm, Thanks for the contribution.

**Pull request merged**